internal/service/networkpeering/resource_network_peering_test.go

```diff
@@ -148,23 +104,28 @@ func TestAccNetworkRSNetworkPeering_basicGCP(t *testing.T) {
 }
 
 func TestAccNetworkRSNetworkPeering_AWSDifferentRegionName(t *testing.T) {
-	acc.SkipTestForCI(t)
 	var (
-		peer                  matlas.Peer
-		resourcePeerName      = "mongodbatlas_network_peering.diff_region"
-		resourceContainerName = "mongodbatlas_network_container.test"
-		projectID             = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
-		vpcID                 = os.Getenv("AWS_VPC_ID")
-		vpcCIDRBlock          = os.Getenv("AWS_VPC_CIDR_BLOCK")
-		awsAccountID          = os.Getenv("AWS_ACCOUNT_ID")
-		containerRegion       = "US_WEST_2"
-		peerRegion            = strings.ToLower(strings.ReplaceAll(os.Getenv("AWS_REGION"), "_", "-"))
-		providerName          = "AWS"
+		vpcID           = os.Getenv("AWS_VPC_ID")
+		vpcCIDRBlock    = os.Getenv("AWS_VPC_CIDR_BLOCK")
+		awsAccountID    = os.Getenv("AWS_ACCOUNT_ID")
+		orgID           = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		containerRegion = "US_WEST_2"
+		peerRegion      = strings.ToLower(strings.ReplaceAll(os.Getenv("AWS_REGION"), "_", "-"))
+		providerName    = "AWS"
+		projectName     = acc.RandomProjectName()
 	)
+	checks := commonChecksAWS(vpcID, providerName, awsAccountID, vpcCIDRBlock, peerRegion)
+	checks = acc.AddAttrSetChecks(resourceNameContainer, checks, "project_id", "atlas_cidr_block", "provider_name", "region_name")
+	mapChecksContainer := map[string]string{
+		"atlas_cidr_block": "192.168.208.0/21",
+		"provider_name":    providerName,
+		"region_name":      containerRegion,
+	}
+	checks = acc.AddAttrChecks(resourceNameContainer, checks, mapChecksContainer)
 
-	resource.Test(t, resource.TestCase{
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
why are we moving this back to Parallel? I lost track with what should be Test and what should be ParallelTest

Reply:
`Test`: Can only run one test at a time
`ParallelTest`: Can run multiple tests in parallel

This test is refactored to create and use its own project instead of reading a project from `MONGODB_ATLAS_PROJECT_ID`, making it safe to run it in parallel.

Reply:
ok thanks, my concern is more around having a precise guideline of what can run in parallel and what not - right now it seems it's on a test-level basis 

cc @lantoli